.gitlab-ci.yml

```diff
@@ -49,6 +62,19 @@ lassen testing:
     strategy: depend
     include: .gitlab/lassen/pipeline.yml
 
+# lassen distconv testing:
+#   stage: run-all-clusters
+#   variables:
+#     JOB_NAME_SUFFIX: _distconv
+#     SPACK_ENV_BASE_NAME_MODIFIER: "-multi-stage-distconv"
+#     SPACK_SPECS: "+cuda +distconv +nvshmem +fft"
+#     WITH_WEEKLY: "${LBANN_CI_RUN_WEEKLY}"
+#     WITH_CLEAN_BUILD: "${LBANN_CI_CLEAN_BUILD}"
+#     TEST_FLAG: "test_*_distconv.py"
+#   trigger:
+#     strategy: depend
+#     include: .gitlab/lassen/multi_stage_pipeline.yml
```

Comment:
What's the issue here? Arguably this is the most important of the test jobs for this PR...